services/workflows-service/src/global.d.ts

```diff
@@ -12,4 +13,8 @@ declare module '@prisma/client' {
   export type WorkflowDefinition = Omit<_WorkflowDefinition, 'config'> & {
     config: WorkflowConfig | any;
   };
+
+  export type Customer = Omit<_Customer, 'subscriptions'> & {
+    subscriptions: TCustomerSubscription | any;
```

Comment:
A union with `any` just results in `any`.

Reply:
i cant tell, as you can see i follow the existing WorkflowDefinition